source/connect/tls-ssl-connection.txt

```diff
@@ -11,3 +11,86 @@ TLS/SSL Connection Tab
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`TLS / SSL` tab allows you to connect deployments via TLS / SSL.
+For more information on :guilabel:`TLS / SSL`, see :manual:`TLS Options 
+</reference/connection-string/#tls-options>`
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`TLS / SSL` tab.
+
+      You have the option to use a ``Default`` connection or to set the TLS / SSL 
+      connection ``On`` or ``Off``. 
+
+      Default
+      ~~~~~~~
+
+      The ``Default`` option does not use the ``TLS / SSL`` connection option. 
+      You can specify additional options when using ``Default``. 
+      
+      TLS / SSL
+      ~~~~~~~~~
+
+      The ``On`` option is enabled when using a :manual:`DNS seedlist (SRV) 
+      </reference/connection-string/#std-label-connections-dns-seedlist>` in 
```

Comment:
Not sure I understand this first sentence. I have a non-DNS seedlist connection string here, but I'm still able to select the On option:

<img width="700" alt="Screen Shot 2022-03-11 at 1 40 30 PM" src="https://user-images.githubusercontent.com/29379027/157931775-05e9eab9-2ada-43f1-ab8f-e7c52cb236e5.png">

Reply:
I added a note to try to clarify this but will ask tech for further help on clarifying.